manifests/t/TravisLane/SteamArtManager/1.4.5/TravisLane.SteamArtManager.locale.en-US.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TravisLane.SteamArtManager
+PackageVersion: 1.4.5
+PackageLocale: en-US
+Publisher: Travis Lane

```

Comment:
```suggestion
Publisher: tormak
```
WinGet relies on metadata from registry to match upgrades and the metadata in the manifest must match whatever is written to registry (that is visible in `appwiz.cpl` / Apps & Features). Since the Publisher value is `tormak` in Apps&Features, it must be the same value written in the manifest

![image](https://user-images.githubusercontent.com/88161975/230727598-10a8b1d3-9c6a-49ae-b39f-6424a4e5fc7e.png)